agent/engine/docker_task_engine.go

```diff
@@ -1954,27 +1954,20 @@ func (engine *DockerTaskEngine) createContainer(task *apitask.Task, container *a
 		} else {
```

Comment:
non blocking: `This is a short term solution only for specific regions` comment is specific to the `else` section for isolated regions as per my understanding. We can consider moving the comment to relevant block.